prod-server-integration-tests/test/admin-web-api.js

```diff
@@ -69,4 +69,46 @@ describe('admin.* Web APIs', function () {
       assert.equal(get3.no_settings_applied.length, userIds.length);
     });
   });
-});
\ No newline at end of file
+
+  describe('admin.auth.policy.{assign|get|remove}Entities', function () {
+    /* 
+      To run this test suite manually, you need an email_password auth policy
+      enabled on the Enterprise Org. For this, you need to have an Enterprise Org
+      with SSO enabled. You will additionally need to export a User ID for 
+      a user managed by the IDP for testing. 
+
+      export SLACK_SDK_TEST_GRID_USER_ID= 
+     */
+    it('should assign an entity', async function () {
+      try {
+        const res = await orgAdminClient.admin.auth.policy.assignEntities({
+          entity_ids: [process.env.SLACK_SDK_TEST_GRID_USER_ID],
+          entity_type: "USER",
+          policy_name: "email_password",
+        });
+        assert.equal(res.ok, true);
```

Comment:
Having the following assertion is a bit easier to check the result than just knowing `false`
```js
assert.isUndefined(res.error);
````

Reply:
I have added these assertions!

Reply:
@srajiang Thanks. Perhaps, you can remove `assert.equal(res.ok, true);` in L88 because the old assertion fails before the newly added one 😸